components/tikv_util/src/metrics/threads_dummy.rs

```diff
@@ -7,8 +7,31 @@ other than Linux. PRs are welcome!
 
 */
 
+use std::collections::HashMap;
 use std::io;
 
 pub fn monitor_threads<S: Into<String>>(_: S) -> io::Result<()> {
     Ok(())
 }
+
+pub struct ThreadInfoStatistics {}
+
+impl ThreadInfoStatistics {
+    pub fn new() -> Self {
+        ThreadInfoStatistics {}
+    }
+
+    pub fn record(
+        &mut self,
+    ) -> (
+        HashMap<String, u64>,
```

Comment:
I don't like returning three hash maps here, how do we do if we want to add another statistic?

Reply:
OK, I'll fix it.